index.js

```diff
@@ -112,7 +102,7 @@ function _init() {
   }
 
   eyes.defaultWindowSize = null;
-  eyes.setBatch(appName, getBatchUUID());
+  eyes.setBatch(batchName, batchId, 0);
```

Comment:
for backward compatibility:

```js
if (process.env.APPLITOOLS_BATCH_ID) {
  eyes.setBatch(null, process.env.APPLITOOLS_BATCH_ID, 0);
} else {
  eyes.setBatch(appName, getBatchUUID());
}
```

and now when I see it like this, two questions:
1. btw, what is the `0` for?
2. why don't we pass the `appName` instead of `null`?

Reply:
oh, now I see that these are the instructions for the branches support where they explicitly ask for the `null` parameter:
https://applitools.com/docs/topics/integrations/github-integration.html?Highlight=branches#Prepare.

anyway, I suggest backward support

Reply:
I'm not sure the backward support here is needed, since it shouldn't have an effect on the tests.
It's just a way to batch the tests.
The `getBatchUUID()` function  generates a random uuid, so I think passing the commit hash instead wont matter.
What i'm not sure at the moment, is if passing the `appName` should be supported as a fallback.
I'll try to investigate